packages/webapp/src/containers/Animals/Inventory/useAnimalInventory.tsx

```diff
@@ -304,7 +304,9 @@ const useAnimalInventory = () => {
     defaultAnimalTypes,
   ]);
 
-  return { inventory, isLoading };
+  const animalsExistOnFarm = inventory.length > 0;
+
+  return { inventory, animalsExistOnFarm, isLoading };
```

Comment:
Can we create a separate hook for `animalsExistOnFarm`?
The formatting and unnecessary API calls feel overwhelming just to know if animals exist...

Reply:
@SayakaOno its a great idea! I have done it andwill push now